how-to-guides/custom-events-pass.ipynb

```diff
@@ -0,0 +1,90 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# How to create Custom Events based on Forward or Backward Pass"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Ignite provides you the flexibility to add events based on the loss\n",
+    "calculation and backward pass as follows:"
```

Comment:
This sounds like custom events helps with loss calculation and backward pass. Please rephrase this sentence as custom events is a generic concept and here it is an example